packages/insomnia-app/app/ui/components/settings/general.js

```diff
@@ -326,10 +371,36 @@ class General extends React.PureComponent<Props, State> {
         </div>
 
         <div className="form-row">
+          <div className="form-control form-control--outlined">
+            <label>
+              Editor Font Weight
+              {fontsMono ? (
+                <select
+                  name="fontMonospaceWeight"
+                  value={settings.fontMonospaceWeight}
+                  onChange={this._handleFontChange}>
+                  {editorFontWeights.map((item, index) => (
+                    <option key={index} value={item}>
```

Comment:
Definitely want to [avoid using index as key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). `item` here should always be unique, so we can key with that.

```suggestion
                  {editorFontWeights.map(item => (
                    <option key={item} value={item}>
```